components/brave_wallet_ui/common/async/base-query-cache.ts

```diff
@@ -333,7 +333,7 @@ export class BaseQueryCache {
     }
     return token.logo.startsWith('ipfs://')
       ? (await this.getIpfsGatewayTranslatedNftUrl(token.logo)) || ''
-      : `chrome://erc-token-images/${token.logo}`
+      : `chrome://erc-token-images/${encodeURIComponent(token.logo)}`
```

Comment:
hey just wanted to double check, but do you think we ever double encode these? There's a lot of mentions of token.logo around.

cc @brave/crypto-wallets-front-end